doc/role-icingaweb2/role-icingaweb2.md

```diff
@@ -29,8 +29,20 @@ icingaweb2_db:
 * `icingaweb2_db_import_schema: boolean`
   * Decides whether the schema should be imported in the database defined at `icingaweb2_db`. **Default: False**
 
-* `icingaweb2_admin_<username|password>: string`
+* `icingaweb2_admin_<username|password|recreate>: string`
   * Set the username and password for the first admin user for Icinga Web 2.
+  * Recreate can be used to change password of admin. **Default:False**
```

Comment:
`icingaweb2_admin_<username|password|recreate>: string` this gives the first impression that the recreate should also be given a string. Just separate them.

Reply:
Is implemented.